bin/concat_prefactor_files

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+"""
+Script to concatenate the MS files from Prefactor for input to Rapthor
+"""
+import argparse
+from argparse import RawTextHelpFormatter
+import glob
+import os
+import sys
+import subprocess
+import casacore.tables as pt
+import numpy as np
+
+
+def main(input_path, output_file):
+    """
+    Concatenate MS files from Prefactor for input to Rapthor
+
+    Parameters
+    ----------
+    input_path : str
+        Full path to the directory with the Prefactor MS files
+    output_file : str
+        Filename of output file
+    """
+    msfiles = glob.glob(os.path.join(input_path, '*.ms'))
```

Comment:
I think MS-files could have both `.ms` and `.MS` as extension. Maybe good to check for both?

Reply:
Yes, good idea. I'll add a check for `.MS`.